connector_jira_tempo/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,14 @@
+* Simone Orsi <simahawk@gmail.com>
```

Comment:
put c2c on top and move me inside c2c pls :)

Reply:
Thank

Reply:
Thank for review